code/queue/Infrastructure/src/main/java/ru/unn/agile/queue/infrastructure/TxtLogger.java

```diff
@@ -0,0 +1,68 @@
+package ru.unn.agile.queue.infrastructure;
+
+import ru.unn.agile.queue.viewmodel.ILogger;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+
+public class TxtLogger implements ILogger {
+    private static final String DATE_AND_TIME_FORMAT = "yyyy-MM-dd HH:mm:ss";
```

Comment:
"yyyy-MM-dd HH:mm:ss.SSS" should be more usefull

Reply:
Done